class_server.js

```diff
@@ -51,12 +51,20 @@ app.post('/classes/create/add', function(req, res) {
 		console.log('Executing command: ' + command);
 		
 		child_process.exec(command, function(error, stdout, stderr) {
-			res.json({
-					error: error,
-					stdout: stdout,
-					stderr: stderr
-			});
+			//res.json({
```

Comment:
Decided not to remove this for the time being. Not quite sure if it was necessary to render JSON to the user